src/components/utility/NavHelper/index.js

```diff
@@ -18,6 +19,14 @@ export default class NavHelper extends Component{
     }
   }
 
+  componentDidMount(){
```

Comment:
This makes sure that clicking 'delete' when targeting the body wouldn't go to previous page.  A few users, including myself, experienced an issue where this would cause them to leave a canvas.  Apparently the area they targeted (while on the spaces show page) was the body.